README.md

```diff
@@ -44,6 +44,26 @@ Detect secrets...........................................................Passed
 ```
 
 
+## Run your first audit
+
+The detect-secrets tool may discover some secrets in your repository that you want to exclude because it's a false positive. It may also find lots of legitimate secrets that you need to confirm.
```

Comment:
Minor grammar thing
"..a secret ... it's a false positive.."
OR 
"..some secrets ... they are false positives.."

Reply:
Done